changelog/v1.19.0-beta1/bump-moby.yaml

```diff
@@ -0,0 +1,7 @@
+changelog:
```

Comment:
actually, I missed that this is merging to 1.18.x branch. So, the changelog should not be in v1.19.0?